std/typecons.d

```diff
@@ -7446,12 +7446,12 @@ template isBitFlagEnum(E)
     {
         enum isBitFlagEnum = (E.min >= 0) &&
         {
-            foreach (immutable flag; EnumMembers!E)
-            {
+            static foreach (immutable flag; EnumMembers!E)
```

Comment:
I feel that this change is a net negative. We don't want a user to have to puzzle out why the double braces are here and what they mean while also trying to learn what `BitFlags` is for and how to use it.

Reply:
This isn't an example. It's the actual implementation. I just spotted a `static foreach` here and included it as a separate commit.